package.json

```diff
@@ -59,7 +59,7 @@
   },
   "dependencies": {
     "algoliasearch": "^3.8.0",
-    "algoliasearch-helper": "^2.5.0",
+    "algoliasearch-helper": "^2.6.1",
```

Comment:
youll have to shrinkwrap.

One easy way to avoid a lot of shrinkwrap changes:

```sh
# remove the package.json change
rm -rf node_modules
npm cache clean
npm install
rm -rf node_modules #yes..
npm install # from cache..
npm install algoliasearch-helper@2.6.1 --save
npm install algoliasearch-helper@2.6.1 --save
npm shrinkwrap
```

Yes that's the way to "upgrade a package when using shrinkwrap", the easiest and safest.